### DISCUSSION TITLE
Making updates on forms doesn't work.. Please help

### DISCUSSION THREAD

**Discussion by bomenzijnchill** [quillforms/quillforms#122] [Q&A] [answered]
Hi guys, 

I'm new here but hopefully someone can help me. 

I'm a longtime (paid) user of Quillforms. Making forms and updating them for my [wordpress website](www.creativebros.nl/tarieven) was never a problem, until now.

Every time I make changes to a form it says "Saved successfully" on the bottom of the screen, but if I refresh the page nothing has been changed...

Video example
https://user-images.githubusercontent.com/132282236/235474473-839497ce-cf8a-4122-bc0f-d464fa2913e2.mp4

What did already do?
- updating everything
- turn on and off pc (kidding)

But I don't know where to start.

Please let me know if you have any idea how to fix this problem.
Thank you!

**Comment by mdmag** [comment]
Hello @bomenzijnchill,

Thanks for reaching out!
Are you using a caching plugin on your WP site?

**Reply by bomenzijnchill** [reply]
Hi! 


Good thinking, I used to have that but now I don't use that anymore. But I do use Wordfence, also turned it off to check if maybe a firewall was interrupting. But it didn't cnange anything.

So still having the problem..

**Reply by bomenzijnchill** [reply]
Here are some screenshots of all the plugins i'm using atm

![image](https://user-images.githubusercontent.com/132282236/235635710-3aa066cd-01d6-470d-ac3c-e73d70d8fac8.png)
![image](https://user-images.githubusercontent.com/132282236/235635806-2d4fe3d4-ae11-4e0d-9eb2-9a4954087c41.png)

**Comment by mdmag** [comment]
That's weird.
Do you have the ability to deactivate the plugins one by one to know if there is a conflict with a specific plugin?

**Comment by mdmag** [comment]
Some hosting services offers caching as well without having a plugin for that.

**Comment by mdmag** [comment]
@bomenzijnchill,
If the issue still persists, can you deactivate Quill Forms jump logic addon and test?
Curious to know the root cause for this issue.

**Reply by bomenzijnchill** [reply]
I’ll try some stuf today and let you know!

**Comment by bomenzijnchill** [comment, marked as answer]
Hi @mdmag

I think I found the problem. When I deactivated the "Make.com" plugin it worked again.. Very weird

So for now it seems to be fixed. Luckily I don't use this plugin atm.

![Scherm­afbeelding 2023-05-08 om 16 52 33](https://user-images.githubusercontent.com/132282236/236857237-bdbfd174-7e7b-4f69-98b3-21ea692abe0f.png)

**Reply by bomenzijnchill** [reply]
Also did a double check. When turning it on again Quill forms doesn't work again. So it was definitely the "Make" plugin doing something. 

If you want me to share some more information to potentially fix this, just ask!

**Comment by mdmag** [comment]
@bomenzijnchill 
I will try to download this plugin and test it.
Thanks a lot for your co-operation!
